Instant-Messenger/src/proxy_fe/ProxyFE.cpp

```diff
@@ -69,7 +69,7 @@ void ProxyFE::prepareSocketConnection(int* socket_fd, sockaddr_in* serv_addr)
     }
 
     // Forcefully attaching socket to the port
-    if (setsockopt(*socket_fd, SOL_SOCKET, SO_REUSEADDR | SO_REUSEPORT, &opt, sizeof(opt))) 
+    if (setsockopt(*socket_fd, SOL_SOCKET, SO_REUSEADDR, &opt, sizeof(opt)))

```

Comment:
Use as it was before

Reply:
It does not work on Macos.

Reply:
Perioed. We need to either use SO_REUSEADDR or SO_REUSEPORT. Based on my researches, SO_REUSEADDRis advised.